docs/index.html

```diff
@@ -11,23 +11,16 @@
     <section class="displayBoardSection flexbox">
       <h1>Leaderboard</h1>
       <article class="scoreBoardArticle">
-        <h2>Recent scores <button type="button">Refresh</button></h2>
+        <h2>Recent scores <button type="button" class="refresh">Refresh</button></h2>
         <ul class="scoreBoard">
-          <li>Name: 100</li>
-          <li>Name: 100</li>
-          <li>Name: 100</li>
-          <li>Name: 100</li>
-          <li>Name: 100</li>
-          <li>Name: 100</li>
-          <li>Name: 100</li>
         </ul>
       </article>
     </section>
     <section class="addScoreSection flexbox">
       <h2>Add your score</h2>
       <form class="addScore flexbox">
-        <input type="text" placeholder="Your name">
-        <input type="text" placeholder="Your score">
+        <input type="text" class="name" placeholder="Your name">
+        <input type="text" class="score" placeholder="Your score">
```

Comment:
- [ ] [OPTIONAL] Kindly make sure that the `score` input can only accept numeric values as now I can add a score as a regular text.

Reply:
ok